ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -110,6 +660,8 @@
         }
         function onInit(e) {
             let formContainer =  e.detail;
+            formContainerGlobal = formContainer;
+            initializeHamburgerMenu();
```

Comment:
Can we move the entire hamburger logic to a separate class, so that there is a clear code segregation. Also, there could be customers with custom form container component, having a separate class would also mean they could integrate the changes in their code easily

Reply:
Moved to a separate class.